graph-http/src/intoresponse.rs

```diff
@@ -179,8 +185,29 @@ impl<'a, T> IntoResponseBlocking<'a, T> {
         let headers = response.headers().clone();
         let status = response.status();
         let url = GraphUrl::from(response.url());
-        let json = response.json().map_err(GraphFailure::from)?;
-        Ok(GraphResponse::new(url, json, status, headers))
+        let mut json: U = response.json().map_err(GraphFailure::from)?;
+        let mut values: Vec<V> = vec![];
+
+        if self.client.client.borrow().follow_next_links {
+            loop {
+                values.append(&mut json.value());
+                match json.next_link() {
+                    Some(next_link) => {
+                        let url = GraphUrl::parse(&next_link)?;
+                        self.client.set_url(url);
+                        let response = self.client.response()?;
+                        json = response.json().map_err(GraphFailure::from)?;
+                    }
+                    None => {
+                        break;
+                    }
+                }
+            }
+            json.value().append(&mut values);
+            Ok(GraphResponse::new(url, json, status, headers))
```

Comment:
One of my concerns about this is that there may be hundreds to thousands of next link calls which will cause a very long delay, blocking the thread for an unknown time, and making it difficult to use effectively in production environments in those cases. We should add a doc comment over this method to tell the caller that this method could result in a long running operation when there are many next link calls.